shared/src/androidMain/kotlin/co/touchlab/kampstarter/KoinAndroid.kt

```diff
@@ -25,4 +27,7 @@ actual val platformModule: Module = module {
             context.getSharedPreferences("KAMPSTARTER_SETTINGS", Context.MODE_PRIVATE)
         AndroidSettings(preferences)
     }
+
+    val baseKermit = Kermit(LogcatLogger())
+    factory { (tag: String?) -> baseKermit.withTag(tag ?: "KampKit") }
```

Comment:
This will create a new kermit instance every time we don't pass a tag. What about this instead?
```suggestion
    val baseKermit = Kermit(LogcatLogger()).withTag("KampKit")
    factory { (tag: String?) -> if (tag != null) baseKermit.withTag(tag) else baseKermit }
```